src/routez/router.zig

```diff
@@ -121,6 +121,10 @@ pub fn match(
     var len: usize = undefined;
 
     inline for (route.path) |c, i| {
+        comptime if (state == .Start and c == '*') {
```

Comment:
Move this inside the switch in the `.Start` after `'/'`.

Reply:
I thought I tried that and had some issue. I'll give it a another shot.

Reply:
Right - this is my knowledge of the language (or lack thereof). I wanted to bail out at start if it saw the '*' character, but break inside a switch I think just breaks out of the switch, not the whole for loop.

Reply:
Breaking out of a switch is not a thing in Zig, you would need to use a labeled block for that.